CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to Libero
+We encourage contributions of any kind.  By participating in this project, you agree to abide by the [code of conduct](https://libero.pub/code-of-conduct/).
+
+Contributions of code or documentation are covered by our [Contributor License Agreement (CLA)](https://libero.pub/governance/cla/). Please [sign the CLA](https://libero.pub/sign-cla) or ensure your employer has signed the CLA on your behalf.
+
+All Libero repositories will have further instructions in their individual `CONTRIBUTING.md` with more specific details.
+
+# Code Contributions
+The basic steps for code contributions are:
+1. Fork, then clone the repo
+2. Set up your machine and install dependencies
+3. Run all tests and ensure they pass
+4. Make your change, adding new tests
+5. Run all tests and ensure they pass
+6. Push to your fork and make a pull request
```

Comment:
This is a simple process for small contributions and fixes, but it would be helpful to add a disclaimer about bigger contributions as they are better off going through an issue to get design and architectural feedback.